Documentation/kube-flannel.yml

```diff
@@ -0,0 +1,74 @@
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: kube-flannel-cfg
+  namespace: kube-system
+  labels:
+    tier: node
+    app: flannel
+data:
+  cni-conf.json: |
+    {
+      "name": "cbr0",
+      "type": "flannel",
+      "delegate": {
+        "isDefaultGateway": true
+      }
+    }
+  net-conf.json: |
+    {
+      "Network": "10.244.0.0/16",
```

Comment:
They should match.


On Aug 27, 2016 2:50 AM, "Deyuan Deng" <notifications@github.com> wrote:

In Documentation/kube-flannel.yml
<https://github.com/coreos/flannel/pull/483#discussion_r76513697>:

> +  namespace: kube-system
> +  labels:
> +    tier: node
> +    app: flannel
> +data:
> +  cni-conf.json: |
> +    {
> +      "name": "cbr0",
> +      "type": "flannel",
> +      "delegate": {
> +        "isDefaultGateway": true
> +      }
> +    }
> +  net-conf.json: |
> +    {
> +      "Network": "10.244.0.0/16",

What's the relationship between Network and cluster-cidr flag in controller
manager? Do they have to match each other?

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/coreos/flannel/pull/483/files/f1cf895dfd2433cd98b9316729c25a9693f5655b..f52fee7cfe1824e959e42bc439088af752dc06bd#r76513697>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/ABtFIV2irSGRbOO8_P5pcKykjVI5nYwRks5qkAhUgaJpZM4JdAh->
.